packages/terra-content-container/src/ContentContainer.jsx

```diff
@@ -50,23 +58,34 @@ const ContentContainer = ({
   fill,
   scrollRefCallback,
   setFocusOnContainer,
+  backgroundColor,
   ...customProps
 }) => {
+  const theme = React.useContext(ThemeContext);
+
   const contentLayoutClassNames = cx([
     `content-container-${fill ? 'fill' : 'static'}`,
     customProps.className,
   ]);
 
+  const background = backgroundColor || 'light';
+  const scrollClassNames = cx(
+    'normalizer',
+    theme.className,
+    { 'content-container-focused-padding': setFocusOnContainer },
+    { dark: background === 'dark' },
+    { light: background === 'light' },
+  );
```

Comment:
Is there a reason we can't do this?
```suggestion
  const scrollClassNames = cx(
    'normalizer',
    theme.className,
    { 'content-container-focused-padding': setFocusOnContainer },
    backgroundColor
  );
```

Reply:
Updated to background